force-app/main/default/classes/ODRIntegrationTest.cls

```diff
@@ -242,6 +242,8 @@ public class ODRIntegrationTest {
         System.assertEquals(oopY, 'B1');
         String oopNu = ODRIntegration.getPractitionerCode('OOP Provider - Nunavit');
         System.assertEquals(oopNu, 'C1');
+        String psychiatricNurse = ODRIntegration.getPractitionerCode('Registered Psychiatric Nurse');
+        System.assertEquals(psychiatricNurse, 'Y9');
```

Comment:
Missing msg parameter in System assertEquals.
https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_class_System_Assert.htm
Also if this class is ver 57.0, then it would be great to switch to the new assert statements.
`Assert.areEqual('psychiatricNurse', 'Y9', 'Practitioner code not matching Y9');`